src/MyElefant.php

```diff
@@ -150,7 +157,7 @@ public function sendSms(array $contacts, $sendDate=null, $message = null, $sende
             $client = new Client();
             $response = $client->request('POST', $this->yamlDatas['URL_MYELEFANT_API'].$this->yamlDatas['URL_MYELEFANT_API_CREATE_CAMPAIGN'],
                 [ 
-                'headers' =>['Authorization' => $this->token, 'Content-Type'=>'application/json'],
+                'headers' =>['Authorization' =>$this->getAuthentification(getenv('SECRET_KEY')), 'Content-Type'=>'application/json'],
```

Comment:
It's a good practice to call `getenv` as early as possible in the code, because it makes it easier to debug and to test.
Even better if you can decouple it completely from the class, by requiring its value in the constructor.